plugins/vlc-video/vlc-video-source.c

```diff
@@ -725,6 +733,8 @@ static obs_properties_t *vlcs_properties(void *data)
 	dstr_free(&filter);
 	dstr_free(&exts);
 
+	obs_properties_add_int(ppts, S_NETWORK_CACHING, T_NETWORK_CACHING, 100, 3600000, 10);
```

Comment:
This line goes over the 80 character limit. Insert a line break after "100," (right after the comma), delete the proceeding space character (right before "3600000"), and insert two tabs to indent.